joomla/joomla-cms
# [4.1] Align the inline help button to the rest of the btns

Pull Request for Issue # .

### Summary of Changes
The static assets **should only be added inside a layout**. The inlinehelp toolbar button is adding the assets (a js file) inside the logic and this makes it impossible to override/remove the script in an easy fashion (eg by editing the relative layout). Also it's pretty obvious that this is wrong as it's the only button that doesn't adhere to the CMS' norm (add the assets only inside the layouts). 


### Testing Instructions
Apply the PR and check that the inlinehelp still works as before (check the global options)


### Actual result BEFORE applying this Pull Request



### Expected result AFTER applying this Pull Request

No assets added in the logic
![Screenshot 2022-03-23 at 14 09 19](https://user-images.githubusercontent.com/3889375/159706859-2206f8f2-8e3c-4da4-8b97-2a7be372a810.png)

All the assets added in the layouts
![Screenshot 2022-03-23 at 14 08 47](https://user-images.githubusercontent.com/3889375/159706954-9f4a3668-e8d8-46ba-aed8-106ea36b306e.png)

### Documentation Changes Required

No, but it would be good to pull a flag whenever a PR is adding static assets in places other than the layouts
